kedro/extras/transformers/__init__.py

```diff
@@ -1,6 +1,15 @@
 """``kedro.extras.transformers`` is the home of Kedro's dataset transformers."""
+import warnings
 
 from .memory_profiler import ProfileMemoryTransformer
 from .time_profiler import ProfileTimeTransformer
 
 __all__ = ["ProfileMemoryTransformer", "ProfileTimeTransformer"]
+
+warnings.simplefilter("default", DeprecationWarning)
+
+warnings.warn(
+    "Support for transformers will be deprecated in Kedro 0.18.0. "
+    "Please use Hooks `before_dataset_loaded` or `after_dataset_loaded` instead.",
```

Comment:
I guess users could also use `before_dataset_saved` and `after_dataset_saved`.

Reply:
Ah yes good catch. Fixed now! 👌